### ISSUE TITLE
Disable system out for dropped connections

### ISSUE THREAD

**Issue by qwwdfsad** [ktorio/ktor#545]
At least conditionally, otherwise any benchmarking session hangs after a single-shot run and prints every connection to `System.out`

```
2018-08-28 12:43:54.623 [engine-thread] ERROR Application - 200 OK: GET - /
java.io.IOException: Connection reset by peer
        at sun.nio.ch.FileDispatcherImpl.write0(Native Method)
        at sun.nio.ch.SocketDispatcher.write(SocketDispatcher.java:47)
        at sun.nio.ch.IOUtil.writeFromNativeBuffer(IOUtil.java:93)
        at sun.nio.ch.IOUtil.write(IOUtil.java:51)
        at sun.nio.ch.SocketChannelImpl.write(SocketChannelImpl.java:471)
        at io.ktor.network.sockets.CIOWriterKt$attachForWritingDirectImpl$1$1.doResume(CIOWriter.kt:73)
        at kotlin.coroutines.experimental.jvm.internal.CoroutineImpl.resume(CoroutineImpl.kt:42)
        at kotlinx.coroutines.experimental.DispatchedTask$DefaultImpls.run(Dispatched.kt:150)
        at kotlinx.coroutines.experimental.io.internal.MutableDelegateContinuation.run(MutableDelegateContinuation.kt:14)
        at io.ktor.network.util.IOCoroutineDispatcher$IODispatchedTask.run(IOCoroutineDispatcher.kt)
        at io.ktor.network.util.IOCoroutineDispatcher$IOThread$run$1.doResume(IOCoroutineDispatcher.kt:73)
        at kotlin.coroutines.experimental.jvm.internal.CoroutineImpl.resume(CoroutineImpl.kt:42)
        at kotlinx.coroutines.experimental.DispatchedTask$DefaultImpls.run(Dispatched.kt:150)
        at kotlinx.coroutines.experimental.DispatchedContinuation.run(Dispatched.kt:14)
        at kotlinx.coroutines.experimental.EventLoopBase.processNextEvent(EventLoop.kt:136)
```

**Comment by snackycracky** [comment]
where can I find this CIO issue ?

**Comment by ratiofu** [comment]
Is this it? #1119

**Comment by snackycracky** [comment]
this ticket points to here. Cyclic dependency ticketwise ;)

**Comment by oleg-larshin** [comment]
Please check the <a href="https://youtrack.jetbrains.com/issue/KTOR-753">following ticket</a> on YouTrack for follow-ups to this issue. GitHub issues will be closed in the coming weeks.

**Comment by e5l** [comment]
Fixed since 1.4.0
